tests/index.js

```diff
@@ -403,3 +403,109 @@ describe('IntlMessageFormat', function () {
         });
     });
 });
+
+describe('ShortNumberFormat', function () {
+
+  describe('and short number formatting', function () {
+      var msg = '' +
+          'I have {numPeople, shortNumber}';
+
+      var msgFmt = new IntlMessageFormat(msg, 'en-US');
+
+      it('should not format 0', function () {
+          var m = msgFmt.format({
+              numPeople: 0
+          });
+
+          expect(m).to.equal('I have 0');
+      });
+
+      it('should not format < 1000', function () {
+          var m = msgFmt.format({
+              numPeople: 999
+          });
+
+          expect(m).to.equal('I have 999');
+      });
+
+      it('should format 1000', function () {
+          var m = msgFmt.format({
+              numPeople: 1000
+          });
+
+          expect(m).to.equal('I have 1K');
+      });
+
+      it('should format 19000', function () {
+          var m = msgFmt.format({
+              numPeople: 19000
+          });
+
+          expect(m).to.equal('I have 19K');
+      });
+
+      it('should format -1000', function () {
+          var m = msgFmt.format({
+              numPeople: -1000
+          });
+
+          expect(m).to.equal('I have -1K');
+      });
+
+      it('should format 1501', function () {
+          var m = msgFmt.format({
+              numPeople: 1501
+          });
+
+          expect(m).to.equal('I have 2K');
+      });
+
+      it('should format -1501', function () {
+          var m = msgFmt.format({
+              numPeople: -1501
+          });
+
+          expect(m).to.equal('I have -2K');
+      });
+  });
+
+  describe('and short number formatting with options', function () {
+      var msg = '' +
+          'I have {numPeople, shortNumber, options}';
+
+      var msgFmt = new IntlMessageFormat(msg, 'en-US', {
+        shortNumber: {
+            significantDigits: 1
+        }
+      });
```

Comment:
@jasonmit How it would work in ember-intl.  Lmk what you think.  I believe this PR good to go but would love some more feedback if need be! 

```
this.intl.t('people.number', 
  { numPeople: 1200 }, 
  { shortNumber: { significantDigits: 1 } }
)
```

Reply:
```js
this.intl.t('people.number', 
  { numPeople: 1200 }, 
  { shortNumber: { significantDigits: 1 } }
)
```

isn't something that would work out of the box today.  `t()` takes two args: the translation key and the options.  I think the formatter options `shortNumber` would need to be set via named formats in ember-intl land vs supplying a 3rd argument.

Reply:
@jasonmit 😮 I was basing off [this](https://github.com/ember-intl/ember-intl/blob/master/docs/ember-service-api.md#methods) section in the docs with the 3rd arg as `optionalFormats:Object`.  I thought this would work but haven't verified manually.

```
people: '{numPeople, shortNumber} will be attending.'
```

If this doesn't work, lmk how I can `set via named formats in ember-intl land`.  I should build this branch and test out in ember-intl directly sometime this week.

Reply:
Yikes, I'll update the docs to remove the 3rd arg since it does not exist.  Sorry for the troll.

>  lmk how I can set via named formats in ember-intl land

Example of defining a named format:
https://github.com/ember-intl/ember-intl/blob/master/tests/dummy/app/formats.js

Reply:
Those are per user defined right?  Like what needs to happen on the ember-intl side?

Reply:
> Those are per user defined right? Like what needs to happen on the ember-intl side?

Exactly